### PR TITLE
Add helper to set_annotations

### LINKED ISSUE #15: [beta] Add detector annotations
Add a beta helper function to add annotations to a detector

### PR DESCRIPTION
**Pull request by restaste**
Closes #15

### CODE REVIEW

File: src/picterra/client.py [hunk 1]
```diff
@@ -77,25 +96,7 @@ def upload_raster(self, filename, name):
...
-                return False
-
-            if resp.json()['status'] == 'ready':
-                return True
-            elif resp.json()['status'] == 'failed':
-                raise APIError(resp.text)
-            else:
-                return False
-        _poll_at_interval(_is_ready, poll_interval)
+        self._wait_until_operation_completes(resp.json())
```
Comment (julienr) [inline src/picterra/client.py:99]: The commit endpoint for raster now returns an operation ?

File: src/picterra/client.py [hunk 2]
```diff
@@ -155,25 +156,7 @@ def set_raster_detection_areas_from_file(self, raster_id, filename):
...
```
Comment (julienr) [inline src/picterra/client.py:159]: Same question as for rasters, this is now returning an operation ?

**Review by julienr — COMMENTED** [review]
Two questions, otherwise LGTM

File: src/picterra/client.py (hunk 1 above)
Comment (restaste) [reply src/picterra/client.py:99]: Yes, well that was in https://gitlab.com/picterra/platform/-/merge_requests/2232, so it's going to be live next week.

File: src/picterra/client.py (hunk 2 above)
Comment (restaste) [reply src/picterra/client.py:159]: Yes, see answer above.

File: src/picterra/client.py (hunk 1 above)
Comment (julienr) [reply src/picterra/client.py:99]: Thanks !

**Review by julienr — APPROVED** [review]
LGTM. You have to rebase locally apparently

**Pull request merged**